src/main/java/model/Rank.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(6, new Money(new BigDecimal(2000000000))),
```

Comment:
```suggestion
    FIRST(6, new Money(new BigDecimal(2_000_000_000))),
```
숫자형의 경우 이런식으로 ``_``을 추가해도 동일한 값으로 봅니다. 
코드의 가독성을 높이는데 도움이 되겠죠!? 😉

Reply:
피드백 감사합니다. 유념하도록 하겠습니다.